### ISSUE TITLE
Add Content Moderation Notifications module to GovCMS8

### ISSUE THREAD

**Issue by pameeela** [govCMS/govcms8-installer#431]
https://www.drupal.org/project/content_moderation_notifications

** What value does this module/package add to GovCMS?
Provides email notification functionality for workflow transitions. 

** Please provide a brief outline of what this module does.
Allows notifications to be sent to all users of a particular role, or to the content's author when a piece of content is transitioned from one state to another via core's content_moderation module.

** Who does this module benefit:
[ ] end users
[x] content editors
[ ] site builders
[ ] themers
[ ] developers
 
** How could you provide/replicate the functionality of this module using alternative methods, eg in your theme?
You cannot replicate it.

** If this module styles or alters HTML or JavaScript output, can the functionality be provided via the theme? What alternatives have you considered.
No alterations.

** What is the maintenance and support status of the module. Describe the issue queue activity.
Actively supported and under active development. Most open issues are feature requests.

** What permissions are needed to utilise the module (and are any new permissions provided by the module)?
'Administer content moderation notifications' is a new permission provided by this module.

** Does the module modify the database structure and/or store additional metadata on nodes or other entities? If so, why? What are the risks for future updates?
No

** Is the module designed to capture anonymous user data?
No

** Is the output of the module typically fully cacheable? Would the inclusion of this module potentially render pages uncacheable.
Does not produce page output.

** What is your assessment of the quality of this module, the contribution history of the module's maintainers, and the uptake of the module within the Drupal community?
This is a simple, high-quality module, developed and maintained by a well-known Drupal contributor in jhedstrom. The module is stable and recently had a new release in Dec 2019.

** Additional information
This is a feature that we are asked about on pretty much every SaaS project. In D7 we achieved this via Rules, which is not included in D8 so there is currently no way to provide this functionality.

**Comment by sukhrajghuman** [comment]
Our team would also find this very useful !

**Comment by simesy** [comment]
I agree with the summary report from @pameeela. The module looks quality to me and appears to have good test coverage. Low impact on security and performance. if it broke it doesn't bring down the whole site. I would also rather a very fit-for-purpose solution like this, rather than the "rules can do anything" approach.

**Comment by brett-sargeant** [comment]
agree, this would be useful.

**Comment by Belhendo** [comment]
Hi all

Thanks for your suggestion and interest, we're currently considering it, I'll be back in touch soon.

**Comment by pameeela** [comment]
Any update on this? It has come up again from another agency seeking this feature.

**Comment by Belhendo** [comment]
Hi @pameeela 

This is still in our backlog to be looked at. Unfortunately we haven't had the resources to be able to progress it. 

Thanks

Belinda

**Comment by akajains** [comment]
Find it useful. Any progress in this front?

**Comment by isaactownsend** [comment]
Hi,

In light of Rules still being in alpha and blocked (https://github.com/govCMS/GovCMS8/issues/420) and having limited options for notifications would we be able to get an ETA on having this module reviewed or where it sits in terms of priority?

Thank you,
Isaac.

**Comment by catwells** [comment]
Putting our backing behind this request too please. Requested enhancements to one of our departmental managed sites needs this feature. Any estimate on timeframes for including in D9 would be great.
Thanks
Cathy
